jbpm-persistence/jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/timer/GlobalJpaTimerJobInstance.java

```diff
@@ -159,23 +159,18 @@ protected boolean hasEnvironmentEntry(Environment environment, String name, Obje
     }
     
     protected TransactionManager startTxIfNeeded(Environment environment) {
-
-    	try {	    	
-	    	if (hasEnvironmentEntry(environment, "IS_TIMER_CMT", true)) {
-        		return null;
-        	}
-    		if (environment.get(EnvironmentName.TRANSACTION_MANAGER) instanceof ContainerManagedTransactionManager) {
-    			TransactionManager tm = TransactionManagerFactory.get().newTransactionManager();
-    			
-    			if (tm.begin()) {    			
-    				return tm;
-    			}
-    		}
-	    	
+    	try {	
+    	    boolean isTimerCMT = hasEnvironmentEntry(environment, "IS_TIMER_CMT", true);
```

Comment:
A ContainerManagedTransactionManager always return false, so this is an example of hidden dead code. 
I was temptep to return null always, buts lets check what ocurrs is we led the method to do what is originally supposed to do, try to open a new transaction if CMT is false